src/ripple/app/main/Main.cpp

```diff
@@ -342,6 +346,9 @@ int run (int argc, char** argv)
     ("nodetoshard", "Import node store into shards")
     ("replay","Replay a ledger close.")
     ("start", "Start from a fresh Ledger.")
+    ("vacuum", po::value<std::string>(),
+        "VACUUM the transaction db. Optional string argument specifies "
+        "temporary directory path.")
```

Comment:
The parameter to vacuum is not optioal, it's required.

Reply:
fixed